tests/utils/object_store/test_oci_object_store.py

```diff
@@ -34,59 +41,135 @@ def test_oci_obj_store(mock_bucket_name, monkeypatch):
     return oci_os
 
 
-def test_upload_object(test_oci_obj_store, monkeypatch, tmp_path, mock_bucket_name):
-    pytest.importorskip('oci')
+@pytest.mark.skipif(not _OCI_INSTALLED, reason='You must have oci installed to run this test')
```

Comment:
we usually do this by just `oci = pytest.importorskip(oci)` in the test itself

Reply:
ok switched it back